Apps/WebClient/src/ClientApp/src/models/noteTimelineEntry.ts

```diff
@@ -23,7 +23,7 @@ export default class NoteTimelineEntry extends TimelineEntry {
     }
 
     public filterApplies(filterText: string, filterTypes: string[]): boolean {
-        if (!filterTypes.includes("Note")) {
+        if (filterTypes.length > 0 && !filterTypes.includes("Note")) {
```

Comment:
I don't think this is a good idea. That moves the logic of the filters to the individual entries. Instead, Let's move that to the timeline, the same way as the initializeFilters works.

Reply:
@FuriousLlama can you please expand it as to me the implementation above seems to be simpler and less code changes right?

Reply:
The problem is that the logic needs to be maintained in more places. Also moves the logic to each individual entry. Instead, the entry filter logic should be extremely simple